executor/join.go

```diff
@@ -277,6 +290,28 @@ func (e *HashJoinExec) fetchInnerRows(ctx context.Context, chkCh chan<- *chunk.C
 	}
 }
 
+// evalRadixBitNum evaluates the radix bit numbers.
+func (e *HashJoinExec) evalRadixBitNum() {
+	sc := e.ctx.GetSessionVars()
+	// Calculate the bit number needed when using radix partition.
+	if sc.EnableRadixJoin {
```

Comment:
Shouldn't it be `!sc.EnableRadixJoin`?

Reply:
Yes.
The test result in the description is still valid
since I did not set `tidb_enable_radix_join` to true during testing...